.travis.yml

```diff
@@ -1,16 +1,17 @@
 sudo: false
 language: python
 python:
-  - "3.4"
+  - 3.4
 install:
-  - "pip install -r requirements/dev.txt"
-  - pip install coveralls==0.4.4
+  - pip install -U pip wheel
+  - pip install -r requirements/dev.txt
+  - pip install coveralls
 env:
   secure: "MU+x68snHYqrm/aYHgdJcutZoTiRo1sl4MU8K09GfMp/Y4s6McNllqLheg8hM6MZAlQcOTPyp/kE8nr8cneYExl0tvH4yU6gVIZj9C7oOYJm4Qx++uNRlevbQ2Yo1lkHZnL8iNlXvqsOzxrB9ryxNClyc/M4OS6fQuyXzYcC/Fg="
 before_script:
   - psql -c "CREATE DATABASE systersdb;" -U postgres
 script:
-  - coverage run systers_portal/manage.py test --settings=systers_portal.settings.testing
+  - coverage run --source=systers_portal test --settings=systers_portal.setiings.testing
```

Comment:
ooh good catch on the typo @ana-balica Thanks

I'll be iterating on this for a while I'll ping you when I need someone to review :smile_cat: